src/PetriEngine/options.cpp

```diff
@@ -462,12 +459,12 @@ bool options_t::parse(int argc, const char** argv) {
                 throw base_error("Missing argument to --ltl-por");
             } else if (std::strcmp(argv[i + 1], "classic") == 0) {
                 ltl_por = LTLPartialOrder::Visible;
-            } else if (std::strcmp(argv[i + 1], "reach") == 0) {
-                ltl_por = LTLPartialOrder::AutomatonReach;
-            } else if (std::strcmp(argv[i + 1], "mix") == 0) {
-                ltl_por = LTLPartialOrder::VisibleReach;
             } else if (std::strcmp(argv[i + 1], "automaton") == 0) {
-                ltl_por = LTLPartialOrder::FullAutomaton;
+                ltl_por = LTLPartialOrder::Automaton;
+            } else if (std::strcmp(argv[i + 1], "mix") == 0) {
```

Comment:
@njulrik, why do you compare here against "mix", it is not mentioned in the options anymore?